ovms_env/bin/Activate.ps1

```diff
@@ -0,0 +1,247 @@
+<#
+.Synopsis
+Activate a Python virtual environment for the current PowerShell session.
+
+.Description
+Pushes the python executable for a virtual environment to the front of the

```

Comment:
@Raghavyadav17 please remove unnecessary files - entire venv/rag_env/ovms_env and Dockerfile.ubuntu/config.json/make.save